Add Review PO (#581)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/how-to-review-a-po.md

# Reviewing a Purchase Order

> If you need to approve a PO, Accrualify will send you an email alerting you that a PO is in your queue for review and approval. The email will include an orange button titled “Review PO Request”. Click the orange button and it will direct you to the PO that needs your attention.

Reviewer: ```suggestion
If you need to approve a PO, you'll receive an email from Accrualify alerting you that a PO is in your queue for review and approval. Select **Review PO Request** to open the PO.
```

Author: edits look good here!

> If you need to approve a PO, Accrualify will send you an email alerting you that a PO is in your queue for review and approval. The email will include an orange button titled “Review PO Request”. Click on the orange button and it will direct you to the PO that needs your attention.

Reviewer 2: ```suggestion
If you need to approve a PO, Accrualify will send you an email alerting you that a PO is in your queue for review and approval. The email will include an orange button titled “Review PO Request”. Click the orange button and it will direct you to the PO that needs your attention.
```

![](../../.gitbook/assets/How-to-review-a-PO-1.PNG)

> Once you are directed to the pending PO, please review the following details:

Reviewer: ```suggestion
In the pending PO, review the following details:
```

Author: edit is good here!

![](../../.gitbook/assets/How-to-review-a-PO-2.PNG)

1. **Vendor**: Make sure this is the correct vendor you are requesting a purchase from. If you have outstanding RFPs and unsure which vendor you will select, use the most likely vendor that you will go with. The vendor name can be changed at a later date, if necessary.
2. **Start date and end date**: Make sure that the dates accurately reflect the start and completion dates of the service or project being requested
3. **Department**: Validate the department(s) selected reflect where you believe spend should go
4. **Description**: Make sure that the description includes, at the minimum, the following information:
    * *Type of Service*: Software, consulting spend, contractor etc.
    * *Purpose*: the intention for requesting spend
    * *Type of Spend*: fixed (same amount every invoice) or variable (amount on the invoices will differ depending on number of hours worked or type of project)
    * *Invoice billing frequency*: monthly, quarterly, annually, etc.
5. **Total**: Make sure the total amount is correct based on the amount in the related contract or other legal document and is in-line with spend. 

> If you would like to raise comments or questions to either the requestor or the Finance Team, you may do so by opening the message tool in Accrualify located on the right hand side. In the “Notify user(s) (optional)” field, select or enter the individual’s Mattermost email and hit “Tab” to make sure the email is accepted. Then click on the blue “Send” button. This will alert the appropriate parties of your follow up. 

Reviewer: ```suggestion
If you have comments or questions for the requestor or the Finance Team, open the message tool in Accrualify located on the right hand side of the screen. In the **Notify user(s) (optional)** field select or enter the individual’s Mattermost email. Press Tab on your keyboard to make sure the email is accepted. Then select **Send**.
```

Author: edits are good here!

![](../../.gitbook/assets/How-to-review-a-PO-3.PNG)

> Once you have reviewed the PO and are ready to approve, click the green “Approve” button. 

Reviewer: ```suggestion
Once you've reviewed the PO and are ready to approve, select **Approve**.
```

![](../../.gitbook/assets/How-to-review-a-PO-4.PNG)

> After you “Approve” the PO, the status will be updated from "Open" to “Pending”.

Reviewer: ```suggestion
When the PO has been approved, the status will be updated from **Open** to **Pending**.
```

> After you “Approve” the PO, the status will be updated from Open to “Pending”.

Reviewer 2: ```suggestion
After you “Approve” the PO, the status will be updated from "Open" to “Pending”.
```

![](../../.gitbook/assets/How-to-review-a-PO-6.PNG)

> If you need to reject the invoice, click the orange “Reject” button.

Reviewer: ```suggestion
If you need to reject the PO, select **Reject**.
```

![](../../.gitbook/assets/How-to-review-a-PO-5.png)

> If you reject the invoice, it means you disagree with one or more items on the invoice. When you reject the invoice, please provide the reasons for the reject in the free text box that appears after you click on “Reject and Next”.

Reviewer 2: ```suggestion
If you reject the invoice, it means you disagree with one or more items on the invoice. When you reject the invoice, please provide the reason(s) for the rejection in the free text box that appears after you click “Reject and Next”.
```

> If you reject the invoice, it means you disagree with one or more items on the invoice. When you reject the invoice, please provide the reason(s) for the rejection in the free text box that appears after you click “Reject and Next”.

Reviewer: ```suggestion
If you reject the PO it means you disagree with one or more items listed. When you reject the PO, please provide the reason(s) for the rejection in the free text box that appears after you've selected **Reject and Next**.
```

Author: great catch!

![](../../.gitbook/assets/How-to-review-a-PO-7.png)

> If you have any questions on the PO module, please contact Procurement@mattermost.com. 

Reviewer 2: ```suggestion
If you have any questions on the PO module, you can contact Procurement@mattermost.com. 
```

> If you have any questions on the PO module, you can contact Procurement@mattermost.com. 

Reviewer: ```suggestion
If you have any questions on the PO module, you can contact Procurement@mattermost.com.
```
